### ISSUE TITLE
Question about the learing rate

### ISSUE THREAD

**Issue by wboerdijk** [Ali2500/HODOR#1]
Dear Ali,
thanks for making your inspiring work publicly available!
I have a question about the learning rate you use throughout your experiments. E.g., for pre-training on MS Coco, you specify the `base_lr` after warmup to be 0.0001: https://github.com/Ali2500/HODOR/blob/45da7329e32c8c6e52658f5aeebe625c1fefddef/configs/static_image.yaml#L32 and you decay after 100.000 steps by 0.1. Yet, in the appendix of your paper (S4, Learning rate schedule), you state that the learing rate after warmup is `10e-4` (i.e., `0.001`). Is this a typo, or am I misunderstanding something?
Thanks for your help!

**Comment by Ali2500** [comment]
Hi,

Sorry for the late reply. This is indeed a typo in the supplementary text (thanks for pointing it out). The LR used in the code is the 'correct' one (0.0001).

**Comment by wboerdijk** [comment]
Thanks for the answer!
